docker-compose.yml

```diff
@@ -1,11 +1,50 @@
-version: "3.4"
-
 services:
   cosmos:
+    profiles: [local]
     image: mcr.microsoft.com/cosmosdb/linux/azure-cosmos-emulator:latest
     ports:
       - "8081:8081"
-      - "10250-10255:10250-10255"
+
+  next-app:
+    container_name: next-app
+    build:
+      context: .
```

Comment:
Could the context for this be src/new-client - might speed up the docker build a little bit